BookUniverseProject/BookUniverseProject/wwwroot/css/main.css

```diff
@@ -0,0 +1,101 @@
+body {
+    background-image: url("images/5483995.jpg");
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+
+.main-image-container {
+    background-image: url("images/bookuni.jpg"); 
+    background-size: cover; 
+    background-position: center;
+    height: 48vh;
+    border-radius: 10px; 
+    vertical-align: center;
+}
+
+.site-description {
+    font-size: 1.2rem; 
+    color: #000000;
+    text-align: center;
+}
+
+.container {
+    background-color: rgba(255, 255, 255, 0.8); 
+    height: 5vh;
+}
+
+.main-form {
+    background-color: rgba(255, 255, 255, 0.8); 
+    width: 90%; 
+    max-width: 1300px; 
+    height: 60vh;
+    margin: 10% auto;
+    padding: 3vw;
+    border: 1px solid #ccc;
+    border-radius: 15px;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    color: #fff;
+}
+
+.registration-form {
+    background-color: rgba(255, 255, 255, 0.8);
+    width: 90%; 
+    max-width: 900px; 
+    height: 60vh;
+    margin: 10% auto;
+    padding: 3vw;
+    border: 1px solid #ccc;
+    border-radius: 15px; 
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    color: #fff;
+    font-size: 1.2rem; 
+}
```

Comment:
I believe we should discuss which unit (px or rem) to use so that it's the same everywhere.